In-house patch for Solaris L3 agent driver configuration variables.
This patch is not currently suitable for upstream contribution.

--- openstack-neutron-8.2.0/manifests/agents/l3.pp.orig 2016-08-07 11:31:04.742484834 -0700
+++ openstack-neutron-8.2.0/manifests/agents/l3.pp       2016-08-07 11:34:02.877745750 -0700
@@ -87,6 +87,14 @@
 #   'dvr_snat': enable DVR with centralized SNAT support (DVR for single-host, for testing only)
 #   Defaults to 'legacy'
 #
+# [*ra_confs*]
+#   (optional) Path to store IPv6 RA config files.
+#   Defaults to undef
+#
+# [*allow_forwarding*]
+#   (optional) For the Solaris l3 agent, allow forwarding between tenant's networks.
+#   Defaults to undef
+#
 # [*allow_automatic_l3agent_failover*]
 #   DEPRECATED: Has no effect in this class. Use the same parameter in neutron::server instead.
 #
@@ -132,6 +141,8 @@ class neutron::agents::l3 (
   $ha_vrrp_advert_int               = '3',
   $agent_mode                       = 'legacy',
   $availability_zone                = $::os_service_default,
+  $ra_confs                         = undef,
+  $allowing_forwarding              = undef,
   # DEPRECATED PARAMETERS
   $allow_automatic_l3agent_failover = false,
   $use_namespaces                   = $::os_service_default,
@@ -179,6 +190,8 @@ class neutron::agents::l3 (
     'DEFAULT/agent_mode':                       value => $agent_mode;
     'DEFAULT/network_device_mtu':               value => $network_device_mtu;
     'DEFAULT/use_namespaces':                   value => $use_namespaces;
+    'DEFAULT/ra_confs':                         value => $ra_confs;
+    'DEFAULT/allow_forwarding_between_networks':value => $allow_forwarding_between_networks;
     'AGENT/availability_zone':                  value => $availability_zone;
   }


